### PR TITLE
Add adjusted icons for Intellij

### PR DESCRIPTION
**Pull request by lcadzy**

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/run/SparkBatchJobDebugExecutor.java [hunk 1]
```diff
@@ -44,15 +44,15 @@ public String getToolWindowId() {
...
+        return Optional.ofNullable(StreamUtil.getImageResourceFile("/icons/ToolWindowSparkJobDebug_13x.png"))
                 .map(Icon.class::cast)
                 .orElse(AllIcons.General.Debug);
     }
 
     @NotNull
     @Override
     public Icon getIcon() {
-        return Optional.ofNullable(StreamUtil.getImageResourceFile("/icons/ToolWindowSparkJobDebug.png"))
+        return Optional.ofNullable(StreamUtil.getImageResourceFile("/icons/ToolWindowSparkJobDebug_13x.png"))
```
Comment (wezhang) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/run/SparkBatchJobDebugExecutor.java]: Let's keep this unchanged

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/run/SparkBatchJobRunExecutor.java [hunk 2]
```diff
@@ -42,15 +42,15 @@ public String getToolWindowId() {
...
-        return Optional.ofNullable(StreamUtil.getImageResourceFile("/icons/ToolWindowSparkJobRun.png"))
+        return Optional.ofNullable(StreamUtil.getImageResourceFile("/icons/ToolWindowSparkJobRun_13x.png"))
                 .map(Icon.class::cast)
                 .orElse(AllIcons.Actions.Upload);
     }
 
     @NotNull
     @Override
     public Icon getIcon() {
-        return Optional.ofNullable(StreamUtil.getImageResourceFile("/icons/ToolWindowSparkJobRun.png"))
```
Comment (wezhang) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/run/SparkBatchJobRunExecutor.java]: Let's keep this unchanged

File: Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/common/CommonConst.java [hunk 3]
```diff
@@ -43,7 +43,7 @@
     public static final String EmualtorPath = "/emulator/";
     public static final String EmulatorArchieveFileName = "service.zip";
     public static final String EmulatorSetupScriptFileName = "setup.sh";
-    public static final String SparkFailureTaskDebugIconPath = "/icons/ToolWindowSparkJobDebug.png";
+    public static final String SparkFailureTaskDebugIconPath = "/icons/ToolWindowSparkJobDebug_13x.png";
```
Comment (wezhang) [inline Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/common/CommonConst.java]: `ToolWindowSparkJobRun.png` related defines are missing?

File: Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/common/CommonConst.java [hunk 4]
```diff
@@ -25,7 +25,7 @@
     public static final String ProductIConPath = "/icons/Product.png";
     public static final String RefreshIConPath = "/icons/Refresh.png";
     public static final String BlobFileIConPath = "/icons/BlobFile.png";
-    public static final String HDExplorerIconPath = "HdExplorer.png";
```
Comment (wezhang) [inline Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/common/CommonConst.java]: Won't remove this.

**Comment by Eskibear** [comment]
The Jenkins CI failed a develop build and messed the dependency cache up. It should work now. Remember to merge the resource in private repo together.

**Comment by lcadzy** [comment]
#2165

**Comment by lcadzy** [comment]
#397

**Pull request merged**